### ISSUE TITLE
Links must have discernible text - Static Site (External links)

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5790]
## Page, Area or Component
https://www.simplereport.gov/support/
https://www.simplereport.gov/unsupported-browser/
https://www.simplereport.gov/getting-started/organizations-and-testing-facilities/onboard-your-organization/

## Quick Summary
The `<a>` on the page with the class `external-site-disclaimer-link-alt` does not have discernible text that describes what the link is for only an icon.

### How to Replicate

**/support**

<img width="546" alt="Screenshot 2023-05-12 at 12 50 18" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/7009345c-a4b3-40ba-9aec-d37ffd9fa66a">

<img width="487" alt="Screenshot 2023-05-12 at 12 50 31" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/43dd92dc-b3d6-4eb2-90dd-6c3800b79513">

**/unsupported-browser**

<img width="279" alt="Screenshot 2023-05-12 at 13 00 43" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/221307b9-6482-486e-94a8-8702cd765c4d">

 **/getting-started/organizations-and-testing-facilities/onboard-your-organization/**
![Screenshot 2023-05-15 at 10 43 25 AM](https://github.com/CDCgov/prime-simplereport/assets/103958711/62ec714d-3be0-4ed6-af47-d8a19b4820bf)

## Accessibility Violation

https://dequeuniversity.com/rules/axe/4.7/link-name?application=AxeChrome

## Action Requested

To solve this problem, you need to fix at least (1) of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

## Source Code (optional)

[which file or line is the issue attached to?]: #
